pepper-apis/src/main/java/org/broadinstitute/ddp/model/study/StudyLanguage.java

```diff
@@ -0,0 +1,38 @@
+package org.broadinstitute.ddp.model.study;
+
+public class StudyLanguage {
+    private String languageCode;
```

Comment:
Since this object is returned directly from the route, it means this object will get serialized to JSON. In that case, I tend to prefer adding `@SerializedName()` annotations to these fields to explicitly spell out the JSON property name:
```java
@SerializedName("languageCode")
private String languageCode
```